Source/Languages/Casing/CamelCaseConverter.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="CaseStyleConverter.ts" />
+
+namespace GLS.Languages.Casing {
+    "use strict";
+
+    /**
+     * Converts a name to camelCase.
+     */
+    export class CamelCaseConverter extends CaseStyleConverter {
+        /**
+         * Converts a name to camelCase.
+         * 
+         * @param name   A name to convert.
+         * @returns The name's equivalent in camelCase.
+         */
+        public convert(name: string): string {
+            name = super.convert(name);
+
+            return name[0].toLowerCase() + name.substring(1);
```

Comment:
So this `convert` just calls super.convert, then lowercases the first letter in returns. Does that mean super.convert will return PascalCase, lower cases the first letter, gets camelCase?
If so, does that mean CaseStyleConverter always does PascalCase by default?